cpp/src/arrow/device.h

```diff
@@ -225,7 +258,7 @@ class ARROW_EXPORT MemoryManager : public std::enable_shared_from_this<MemoryMan
 
   /// \brief Wrap an event into a SyncEvent.
   ///
-  /// @param sync_event passed in sync_event from the imported device array.
+  /// @param sync_event passed in sync_event (should be a CUevent*)
```

Comment:
I don't think this comment is accurate if the Device isn't a CUDA device?

Reply:
Gah, I lost track of which file i was in there and put the wrong comment -_-. I'll fix that